src/native/libs/System.Net.Security.Native.Apple/pal_networkframework.m

```diff
@@ -0,0 +1,433 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_networkframework.h"
+#include <Foundation/Foundation.h>
+
+static WriteCallback _writeFunc;
+static ReadCallback _readFunc;
+static StatusUpdateCallback _statusFunc;
+static nw_protocol_definition_t _framerDefinition;
+static nw_protocol_definition_t _tlsDefinition;
+static dispatch_queue_t _tlsQueue;
+static dispatch_queue_t _inputQueue;
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunguarded-availability-new"
+
+PALEXPORT nw_connection_t AppleNetNative_NwCreateContext(int32_t isServer)
+{
+    if (isServer != 0)  // only client supported at this point.
```

Comment:
Nit: "at this point" is a little vague... I initially read that as "at this point based on where we've gotten to in the code" rather than "at this point in time".
```suggestion
    if (isServer != 0)  // the current implementation only supports client
```